### ISSUE TITLE
GML missing srsName for geometries with deep nesting where srsName cannot be added at the top level

### ISSUE THREAD

**Issue by Kate-Lyndegaard** [halestudio/hale#728]
I came across this working on the Statistical Grid SLD, which requires the use of the geometry type PolygonPatch. It is not possible to add srsName to the top level of PolygonPatch directly because it is located several levels deeper in the geometry. I cannot publish the valid, exported GML in haleconnect because it is missing the srsName.

It would be helpful if hale studio could locate at least one instance of srsName in the geometry, so that the geometry will always have a srsName, if available. 

![grafik](https://user-images.githubusercontent.com/36956461/51325776-f8d55a00-1a6d-11e9-8617-53e421fa6534.png)

**Comment by florianesser** [comment]
@Kate-Lyndegaard Are you sure that the `srsName` attribute is allowed in the `PolygonPatch` element? From a superficial check it looks like that `PolygonPatch` does not inherit from `AbstractGeometryType` and, therefore, doesn't have the `SRSReferenceGroup` attribute group. This would need to be checked more thoroughly, though.

**Comment by stempler** [comment]
@florianesser Yes, that's true. Thus it would be good to have the `srsName` and `srsDimension` at level of the `posList` or any other level that supports that kind of information.

**Comment by Kate-Lyndegaard** [comment]
@florianesser @stempler 
When I add `<gml:posList srsName="http://www.opengis.net/def/crs/EPSG/0/4258" srsDimension="2">`
to `posList` my INSPIRE gml, it passes data analysis on haleconnect and the srs is recognized.

**Comment by JohannaOtt** [comment]
This issue also leads to the exported GML file failing the INSPIRE Validator: https://github.com/inspire-eu-validation/community/issues/196
